ios/PurchasesHybridCommon/PurchasesHybridCommon/VirtualCurrencyInfo+HybridAdditions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  VirtualCurrencyInfo+HybridAdditions.swift
+//  PurchasesHybridCommon
+//
+//  Created by Will Taylor on 3/27/25.
+//  Copyright © 2025 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import RevenueCat
+
+internal extension VirtualCurrencyInfo {
+
+    var rc_dictionary: [String: Any] {
```

Comment:
Hmm NABD but I noticed that in Android, the value `Any` is nullable. Should it also be in iOS? or maybe not nullable in Android?

Reply:
Ah, great catch! These should definitely be consistent. The `balance` value in `VirtualCurrencyInfo` is non-optional on both iOS & Android, so I've removed the optionality on the Android side in d2fd6b2d874fc954d3b9d4b2ebf1ada60d0f7411